src/webhooks/README.md

```diff
@@ -3,6 +3,44 @@
 * Webhooks in "production" are deployed to a Google Cloud Run instance, in the project `super-big-data`. Why? (TODO insert why)
 * The webhook points to `https://product-eng-webhooks-vmrqv3f7nq-uw.a.run.app`
 
+## Generic Event Notifier
+
+The folder `generic-notifier` provides a generic webhook which can be used to send messages to Sentry Slack channels and Sentry Datadog. Using this webhook is VERY simple.
```

Comment:
nit, refrain from making judgements about whether the system you built is simple/complex/well designed/etc. 
It does not add much to the content.